miqa/core/tasks.py

```diff
@@ -49,6 +49,21 @@ def _download_from_s3(path: str, public: bool) -> bytes:
     return buf.getvalue()
 
 
+@shared_task
+def reset_demo():
+    if not settings.DEMO_MODE:
+        raise Exception('Demo Mode is false; noop.')
+    try:
+        demo_project = Project.objects.get(name='Demo Project')
+    except Project.DoesNotExist:
+        raise Exception('No Demo Project; noop.')
+    demo_project.import_path = 's3://miqa-storage/miqa.csv'
+    demo_project.export_path = 'samples/demo.json'
+    demo_project.save()
+    import_data(demo_project.id)
```

Comment:
If it's important that this be executed synchronously, maybe add a code comment indicating why.

Naively, I imagine this could have been written as:
```python
import_data.delay(demo_project.id)
```
but I also suspect there could be important reasons to ensure the demo project data gets re-imported ASAP.